packages/aws-cdk-lib/aws-sns/lib/delivery-policy.ts

```diff
@@ -0,0 +1,100 @@
+/**
+ * Algorithms which can be used by SNS to calculate the delays associated with all of the retry attempts between the first and last retries in the backoff phase.
+ */
+export enum BackoffFunction {
+  /** Arithmetic.
```

Comment:
Not the best doc I've ever written :sweat_smile: ,  the best indicator of to what these each of these do that I found was an image at the very bottom of the doc page here: https://docs.aws.amazon.com/sns/latest/dg/sns-message-delivery-retries.html - suggestions welcome.

Reply:
> I would appreciate it if you could consult with the maintainer again.

As in a CDK maintainer?

Reply:
Yes! After my approval, you need to get approval from the CDK maintainer. I would appreciate it if you could kindly check it at that time!

Reply:
Ah right np, I thought you might have meant someone in the SNS team or something :+1:

Reply:
Ref: https://github.com/aws/aws-cdk/wiki/CDK-Community-PR-Reviews